src/VirtoCommerce.Platform.Web/Controllers/Api/SecurityController.cs

```diff
@@ -523,13 +526,32 @@ public async Task<ActionResult> RequestPasswordReset(string loginOrEmail)
                 user = await UserManager.FindByEmailAsync(loginOrEmail);
             }
 
+            if (user == null)
```

Comment:
Add please comment why we return 200 even if the user not found

Reply:
added